ios/FluentUI/Tooltip/TooltipTokenSet.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Design token set for the `Tooltip` control.
+public class TooltipTokenSet: ControlTokenSet<TooltipTokenSet.Tokens> {
+    public enum Tokens: TokenSetKey {
+        /// The color of the body of the tooltip.
+        case tooltipColor
```

Comment:
nit: maybe you could add a new line between cases. It'd improve readability and be consistent with how other controls' TokenSets are commented.